src/routes/components/FormItemPage.tsx

```diff
@@ -181,6 +192,92 @@ export default function FormItemPage() {
             </Sandbox>
 
             <ComponentProperties properties={componentProps} />
+
+            <h2 id="component-examples" className="hidden" aria-hidden="true">Examples</h2>
+
+            <h3 id="component-example-sortable-columns">Slotted Helper Text</h3>
+            <Sandbox skipRender >
+              <GoAFormItem label="First name"
+                            helpText={helptextReactNode}>
+                <GoAInput onChange={noop} value="" name="item" />
```

Comment:
Can you start a new line for each property? Since that is they way the prettier formatting does it. The same is true for the other places that this is done.

Reply:
done. thanks